### DISCUSSION TITLE
How does one import the Theme enum to use dark mode as shown in the styling example?

### DISCUSSION THREAD

**Discussion by sengho66** [hecrj/iced#1509] [General]
In one of the examples called styling, in the main.rs it uses 
`
use iced::theme::{self, Theme};
`

However when I try that in my own program I get
`
unresolved import `iced::theme`
no `theme` in the root
`

**Comment by arjunair86** [comment]
Have seen this issue with other parts of code as well. Last release of iced was done in May, so I am assuming there are some code changes in mainline which are not reflected in release. Just clone the repo and specify the path for dependency instead of release version. It should work.
